tests/shared/audit/10-base-config.rules

```diff
@@ -0,0 +1,13 @@
+## First rule - delete all
+-D
+
+## Increase the buffers to survive stress events.
+## Make this bigger for busy systems
+-b 8192
+
+## This determine how long to wait in burst of events
+--backlog_wait_time 60000
+
+## Set failure mode to syslog
+-f 1
+
```

Comment:
Just for the record, in case audit configuration is updated in the future, can this data be obtained somehow automatically from the audit package? If so how and is it documented somewhere?

Reply:
I started this thing to see how automating updated of content could look like:
https://gitlab.cee.redhat.com/wsato/content-extractor

It probably can be integrated in the utils directory.